source/routing/raptor_api.cpp

```diff
@@ -693,9 +692,15 @@ get_stop_points( const type::EntryPoint &ep, const type::Data& data,
                     ep.streetnetwork_params.max_duration,
                     data.pt_data->stop_point_proximity_list,
                     use_second);
-        LOG4CPLUS_TRACE(logger, "find " << tmp_sn.size() << " stop_points");
+        LOG4CPLUS_TRACE(logger, tmp_sn.size() << " stop_points found");
         for(auto idx_duration : tmp_sn) {
             auto sp_idx = SpIdx(idx_duration.first);
+
+            if (idx_duration.second < ep.streetnetwork_params.min_duration) {
+                // we sometimes want a minimum fallback duration (not to take the car for 1mn for example)
+                LOG4CPLUS_INFO(logger, idx_duration.second << " is not enough, we skip this stop");
```

Comment:
oups i think it should not even pe printed, was for debug.

On Thu, Apr 9, 2015 at 6:52 PM, Guillaume P. <notifications@github.com>
wrote:

> In source/routing/raptor_api.cpp
> <https://github.com/CanalTP/navitia/pull/994#discussion_r28079895>:
>
> >          for(auto idx_duration : tmp_sn) {
> >              auto sp_idx = SpIdx(idx_duration.first);
> > +
> > +            if (idx_duration.second < ep.streetnetwork_params.min_duration) {
> > +                // we sometimes want a minimum fallback duration (not to take the car for 1mn for example)
> > +                LOG4CPLUS_INFO(logger, idx_duration.second << " is not enough, we skip this stop");
>
> INFO is a bit high, I thing it should be DEBUG or TRACE
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CanalTP/navitia/pull/994/files#r28079895>.
>